app/views/permit_hub_mailer/09_announce_maintenance_email.html.erb

```diff
@@ -1,215 +1,41 @@
-                      <a href="https://buildingpermit.gov.bc.ca/contact" target="_blank" style="text-decoration: underline; color: #464341; font-size: 14px; text-align: left; line-height: 20px; margin-right: 20px;">Contact us</a>
-                      <a href="https://buildingpermit.gov.bc.ca/help" target="_blank" style="text-decoration: underline; color: #464341; font-size: 14px; text-align: left; line-height: 20px; margin-right: 20px;">Help</a>
-                      <a href="https://buildingpermit.gov.bc.ca/login" target="_blank" style="text-decoration: underline; color: #464341; font-size: 14px; text-align: left; line-height: 20px; margin-right: 20px;">Login</a>
-                    </p>
-                    <hr style="border: 0; border-bottom: 1px solid #dddddd; margin: 24px 0; border-color: #464341;">
-                    <p style="font-family: 'Open Sans', sans-serif; font-weight: normal; margin: 0; margin-bottom: 24px; color: #464341; font-size: 14px; text-align: left; line-height: 20px;">© Government of British Columbia.</p>
-                  </td>
-                </tr>
-              </table>
-            </div>
-            <!-- END FOOTER -->
-
-            <!-- END CENTERED WHITE CONTAINER -->
-          </div>
+        <!-- START CONTENT ROW -->
+        <td style="font-family: 'Open Sans', sans-serif; font-size: 16px; vertical-align: top;" valign="top">
+          <h1 style="color: #292929; font-family: 'Open Sans', sans-serif; font-weight: 700; line-height: 1.4; text-align: left; font-size: 28px; margin-top: 0; margin-bottom: 48px;">Notice of System Maintenance</h1>
+          <p style="font-family: 'Open Sans', sans-serif; font-size: 16px; line-height: 26px; font-weight: normal; margin: 0; margin-bottom: 24px;">Dear [User @TODO],</p>
+          <p style="font-family: 'Open Sans', sans-serif; font-size: 16px; line-height: 26px; font-weight: normal; margin: 0; margin-bottom: 24px;">
+            Please be advised that the Building Permit Hub is
+            performing
+            <strong>[@TODO system updates or scheduled maintenance] on
+              [@TODO date]. The system will be unavailable from
+              [start time] to [end time]</strong>.
+          </p>
+          <p style="font-family: 'Open Sans', sans-serif; font-size: 16px; line-height: 26px; font-weight: normal; margin: 0; margin-bottom: 24px;">
+            During the maintenance period, there may be temporary
+            interruptions in system access or reduced
+            functionality. Please save any permits in progress
+            before the scheduled maintenance window.  We apologize
+            for any inconvenience this may cause and appreciate
+            your understanding as we work to improve our services.
+          </p>
+          <p style="font-family: 'Open Sans', sans-serif; font-size: 16px; line-height: 26px; font-weight: normal; margin: 0; margin-bottom: 24px;">
```

Comment:
Just confirming if this page is meant to be stubbed as there is some @TODOs

Reply:
These are for future features to be completed later.